search/elastic.py

```diff
@@ -596,6 +601,13 @@ def search(self,
             }
 
         body = {"query": query}
+
+        if sort:
```

Comment:
Do we need to ensure request will be succeed if sort field doesn't exist?

Reply:
I added "ignore_unmapped" option. And request will  be succeed if sort field doesn't exist. This field will be ignored by elastic.